lib/linguist/languages.yml

```diff
@@ -4673,6 +4673,17 @@ Protocol Buffer:
   codemirror_mode: protobuf
   codemirror_mime_type: text/x-protobuf
   language_id: 297
+Protocol Buffer text format:
```

Comment:
**Nitpick:**

```suggestion
Protocol Buffer Text Format:
```

The capitalisation of `samples/Protocol Buffer text format` will need to be updated as well. Make sure your checkout's `core.ignorecsae` setting is disabled:

~~~sh
git config --local core.ignorecase false
~~~

(This has been a pitfall for many a contributor, including myself…)

Reply:
Done.